components/Post.php

```diff
@@ -84,27 +87,25 @@ protected function loadPost()
     {
         $slug = $this->property('slug');
 
-        $post = new BlogPost;
-
-        $post = $post->isClassExtendedWith('RainLab.Translate.Behaviors.TranslatableModel')
-            ? $post->transWhere('slug', $slug)
-            : $post->where('slug', $slug);
+        $post = BlogPost;
```

Comment:
@LukeTowers I've never seen this syntax before - does this still instantiate a `BlogPost` instance?

Reply:
@bennothommo nope 🤦 That's just called "Luke shouldn't be writing code at 1:30am" 😂